exchanges/binance/binance_wrapper.go

```diff
@@ -545,6 +548,20 @@ func (b *Binance) SubmitOrder(s *order.Submit) (order.SubmitResponse, error) {
 	}
 	submitOrderResponse.IsOrderPlaced = true
 
+	// create fee, cost and rate to fill response
+	var fee, cost, priceTemp float64
+	if len(response.Fills) > 0 {
```

Comment:
I have implemented Trades filling, so we can remove from response Rate, Fee and Cost fields since they can be calculated on the client side. Or leave it as you see fit.